staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go

```diff
@@ -92,14 +93,14 @@ func (*fancyResponseWriter) Flush() {}
 func (*fancyResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) { return nil, nil, nil }
 
 func TestConstructResponseWriter(t *testing.T) {
-	actual := decorateResponseWriter(&simpleResponseWriter{}, nil, nil, nil)
+	actual := decorateResponseWriter(context.TODO(), &simpleResponseWriter{}, nil, nil, nil)
```

Comment:
tests should probably use `context.Background()` since we're probably never going to plumb context through here.

Reply:
Changed. The pr might need the lgtm again. Thanks!